### DISCUSSION TITLE
deprecation flag on index entries?

### DISCUSSION THREAD

**Discussion by m-hue** [osmlab/name-suggestion-index#6371] [Ideas]
Hello,

I was wondering if anyone else would find it useful to mark entries as deprecated. When company A buys company B and rebrands all former B locations to A over some sort of transition period, it may be useful to flag B as deprecated in the index (and ideally, add a comment explaining why and a link to a source).
The true value of this would only become evident once editors (ID, RapID, ...?) know to interpret the deprecated flag and start yelling at the user in one way or another (ideally, leveraging the comment mentioned before - e.g. "B has rebranded to A as of xyz date") when stuff is still tagged as B. However, that is only possible if the information that B's days are over is stored somewhere™. 

Essentially, that would allow editors to do something like "we'd really want you to tag this as A, but if you must tag it as B, these are the canonical tags for it".

Thoughts?

**Comment by UKChris-osm** [comment]
I'd agree with something like this, although it is partly catered for with **matchNames** where by if Company A buys Company B and rebrands, then Company B can become a matchName for Company A, which allows anyone who types in or clicks on an entry that's still listed as Company B, to be suggested that Company A is available to use.

But I do think having Company B display a little message about "this brand no longer exists" or "this brand may no longer exist, please check" would be a handy addition to have.
